src/gui/components/wxGameList.cpp

```diff
@@ -689,7 +689,9 @@ void wxGameList::OnContextMenuSelected(wxCommandEvent& event)
 
 			case kContextMenuSaveFolder:
 			{
-				wxLaunchDefaultBrowser(wxHelper::FromUtf8(fmt::format("file:{}", _pathToUtf8(gameInfo.GetSaveFolder()))));
+				// escape any spaces in file path
+				std::string saveFolder = boost::replace_all_copy(_pathToUtf8(gameInfo.GetSaveFolder()), " ", "%20");
+				wxLaunchDefaultBrowser(wxHelper::FromUtf8(fmt::format("file:{}", saveFolder)));
```

Comment:
```suggestion
				// handles any spaces in file path
				wxLaunchDefaultApplication(wxHelper::FromPath(gameInfo.GetSaveFolder()));
```
Confirmed that this also works. Perhaps it should be preferred as it does not require any additional string editing?

Reply:
Probably best to use `wxHelper::FromPath`, as it can then handle _any_ special characters in the string.

Reply:
Updated